### ISSUE TITLE
Additional Data Source: BfArM

### ISSUE THREAD

**Issue by yetzt** [zerforschung/schnelltesttest.de#38]
The [Bundesinstitut für Arzneimittel und Medizinprodukte](https://www.bfarm.de/) also [lists antigen tests](https://antigentest.bfarm.de/ords/f?p=110:100:4361219547772:::::&tz=1:00) and the data appears to be more extensive than the PDF data from PEI.

**Comment by pajowu** [comment]
Yes, but the bfarm uses the sensitivity reported by the manufactures, which differs a lot from the one the PEI measured

**Comment by yetzt** [comment]
I'm not suggesting relying on the reported sensitivity, but making use of the metadata ("Test exists, not verified yet"). From a user perspective it's better to find their test and see that it is not approved yet than not finding a test.
